lib/spack/docs/configuration.rst

```diff
@@ -202,12 +202,12 @@ supply ``-p`` to Spack on the command line, before any subcommands.
 .. _spack-p:
 
 ^^^^^^^^^^^^^^^^^^^
-``spack --profile``
+``spack -p``
 ^^^^^^^^^^^^^^^^^^^
```

Comment:
I actually kind of like keeping the title as the long option -- Was there a problem with that for auto-creating names of sections?